components/net/bluetooth_thread.rs

```diff
@@ -355,16 +391,18 @@ impl BluetoothManager {
             Some(a) => a,
             None => return drop(sender.send(Err(String::from(ADAPTER_ERROR)))),
         };
-        let devices = self.get_and_cache_devices(&mut adapter);
-        if devices.is_empty() {
-            return drop(sender.send(Err(String::from(DEVICE_ERROR))));
+        let session = BluetoothDiscoverySession::create_session(adapter.get_object_path()).ok();
```

Comment:
what if `BluetoothDiscoverySession::create_session(adapter.get_object_path())`  is not ok?